wondrous-app/utils/helpers.ts

```diff
@@ -64,6 +64,7 @@ export const getMentionArray = (content) => {
 };
 
 export const parseUserPermissionContext = (props) => {
+  // TODO add return type here
```

Comment:
What kind of return type?

Reply:
Just should add typing on what the return is